apisupport/apisupport.ant/src/org/netbeans/modules/apisupport/project/universe/ModuleList.java

```diff
@@ -613,7 +613,7 @@ static void scanPossibleProject(File basedir, Map<String,ModuleEntry> entries,
                 } else {
                     // Wildcard. Convert to regexp and do a brute-force search.
                     // Not the most efficient option but should probably suffice.
-                    String regex = "\\Q" + pattern.replaceAll("\\*\\*", "__DBLASTERISK__"). // NOI18N
+                    String regex = "\\Q" + pattern.replace("\\*\\*", "__DBLASTERISK__"). // NOI18N
```

Comment:
This is not the same as the original, right?

Reply:
Correct. It has been reverted